src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,27 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: {
+  title,
+  user,
+  body,
+  comments,
+} }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+    </div>
+
+    <UserInfo user={user} key={user.id} />
```

Comment:
no need to use `key` here, you need key if you render collection and on every top level element you need it, not on children element, for example:

```

>     <div class="collection">
>       <div class="collection__parent">
>         <div class="collection__item" key="itemId">
>           <div class="collection__link"></div>
>           <div class="collection__text"></div>
>         </div>
> 
>         <div class="collection__item" key="itemId">
>           <div class="collection__link"></div>
>           <div class="collection__text"></div>
>         </div>
>       </div>
>     </div>

```

you need key only on collection__item in my example